tests/unit/formats/spec.test.ts

```diff
@@ -0,0 +1,148 @@
+import chalk from 'chalk';
+import dedent from 'ts-dedent';
+import { createEmitter } from '../../../src/Emitter';
+import { SuiteEvents } from '../../../src/Suite';
+import { format } from '../../../src/formats/Spec';
+
+const noColors = new chalk.Instance({ level: 0 });
+
+test('suiteStart', () => {
+  const emitter = createEmitter<SuiteEvents>();
+  const print = jest.fn();
+
+  format(emitter, print, noColors);
```

Comment:
suuper nit: does this kind of setup make sense in a `beforeEach`? or is it good practice to avoid that?

Reply:
That could definitely make sense, and using `beforeEach` is totally fine (no need to avoid that).

My personal preference is to use `beforeEach` sparingly, and mostly for things that are necessary, but not really relevant to the individual tests.

Another technique is to extract a `setup` function. But that wouldn't save us much here.